deployment/deployment.go

```diff
@@ -0,0 +1,1224 @@
+package deployment
```

Comment:
Within other portions of the region api we've distinguished (using separate files), what is a http end point and what is a library call, this would make it easier to know when modifying files what would potentially break a contract in addition to isolating dependencies.  See router/http.go as an example. This is optional, but wanted to give you an FYI.

Reply:
That makes a ton of sense, but I don't think the space and app packages do that so I must have missed it. Will refactor a bit to match that pattern